templates/workspace_index/index.html.j2

```diff
@@ -2,6 +2,10 @@
 
 {% block content %}
 <ul class="index-grid">
+{% if page.docs_content %}
```

Comment:
oh, this needs to be outside the index grid ul (that's why i force pushed, sorry)

Reply:
I missed that you force-pushed! I force-pushed because I had to rebase on main, and didn't realize there had been changes. Made that change again